lib/template/config/puma.rb

```diff
@@ -1,6 +1,6 @@
 require "./config/config"
 
-environment Config.rack_env
+environment Config.pliny_env
```

Comment:
@brandur This might not have the intended effect, see: https://github.com/interagent/pliny/issues/237#issuecomment-169757105